enterprise/actions.py

```diff
@@ -0,0 +1,59 @@
+"""
+Admin actions.
+"""
+from __future__ import absolute_import, unicode_literals
+
+import unicodecsv
+from six import string_types
+
+from django.http import HttpResponse
+
+
+def export_as_csv_action(description="Export selected objects as CSV file", fields=None, header=True):
+    """
+    Return an export csv action.
+
+    Arguments:
+        description (string): action description
+        fields ([string]): list of model fields to include
+        header (bool): whether or not to output the column names as the first row
+    """
+    # adapted from https://gist.github.com/mgerring/3645889
+    def export_as_csv(modeladmin, request, queryset):  # pylint: disable=unused-argument
+        """
+        Export model fields to CSV  # pylint: disable=protected-access.
+        """
+        opts = modeladmin.model._meta
+
+        if not fields:
+            field_names = [field.name for field in opts.fields]
+        else:
+            field_names = fields
+
+        response = HttpResponse(content_type="text/csv")
+        response["Content-Disposition"] = "attachment; filename={filename}.csv".format(
+            filename=str(opts).replace(".", "_")
+        )
+
+        writer = unicodecsv.writer(response, encoding="utf-8")
+        if header:
+            writer.writerow(field_names)
+        for obj in queryset:
+            row = []
+            for field_name in field_names:
+                field = getattr(obj, field_name)
+                if callable(field):
+                    value = field()
+                else:
+                    value = field
+                if value is None:
```

Comment:
@clintonb 

* As you pointed out, empty string would denote both empty string and `None`.
* Some people (including me) prefer viewing CSV in ordinary text editors rather than Microsoft Excel/Libre Office Calc. Empty/null data is harder to read in that case.

These two are weak arguments, but I suppose some extra care for the users justify four lines of code. Unless you've got a stronger argument against it/